Loaded modules path is wrong

I've been playing around with this module and found some errors.

This one occurs when I try to load a test file in grunt. It tries to get the file from the plugin folder.

```
$ grunt
Running "qunitnode:files" (qunitnode) task
Testing test/diff.js Warning: Cannot find module '/Users/leobalter/dev/balterDiff/node_modules/grunt-qunitnode/test/diff.js' Use --force to continue.
```

Below is the code piece of my Gruntfile:

``` js
grunt.initConfig({
    qunitnode: {
        files: [ "test/diff.js" ]
    }
});
```

--- a/tasks/qunitnode.js
+++ b/tasks/qunitnode.js
@@ -137,7 +137,7 @@ module.exports = function( grunt ) {
 		status = { failed: 0, passed: 0, total: 0, runtime: 0 };
 
 		grunt.util.async.forEachSeries( this.filesSrc, function( src, nextStep ) {
-			var testFile = path.join( __dirname, "../" + src );
+			var testFile = path.resolve( src );
 
 			grunt.event.emit( "qunitnode.spawn", src );
 

